setup.py

```diff
@@ -53,6 +53,7 @@
           "parsley",
           "psycopg2",
           "six",
+          "boto3"
```

Comment:
maybe this should be an [optional dependency](https://setuptools.pypa.io/en/latest/userguide/dependency_management.html?highlight=extras_require#optional-dependencies)?

Reply:
@bmrobin you're right! I just pushed this change in my latest commit. I don't really like adding the import inside the conditional, but didn't know where else to put the new module that contains this code (didn't know if the modules or utils directory would be the best place for it). Let me know if you have any other suggestions!